src/player.cpp

```diff
@@ -9111,6 +9106,15 @@ void player::use_fire(const int quantity)
     } else if (has_charges("zweifire_off", quantity)) {
         use_charges("zweifire_off", quantity);
         return;
+    } else if (has_active_bionic("bio_tools") && ( power_level > (quantity * 5) )) {
+        charge_power( -(quantity * 5) );
```

Comment:
Btw. Why all the redundant parentheses? The inner pair here is redundant. And in the line above, `power_level > quantity * 5` is enough as well (`*` binds stronger than `>`, which binds stronger than `&&`).